src/driver/postgres/PostgresQueryRunner.ts

```diff
@@ -849,9 +849,9 @@ where constraint_type = 'PRIMARY KEY' and tc.table_catalog = '${this.dbName}'`;
      */
     protected buildCreateColumnSql(column: ColumnSchema, skipPrimary: boolean) {
         let c = "\"" + column.name + "\"";
-        if (column.isGenerated === true) // don't use skipPrimary here since updates can update already exist primary without auto inc.
+        if (column.isGenerated === true && column.type !== "uuid") // don't use skipPrimary here since updates can update already exist primary without auto inc.
             c += " SERIAL";
-        if (!column.isGenerated)
+        if (!column.isGenerated || (column.isGenerated === true && column.type === "uuid"))
```

Comment:
actually this expression can be simplified to `if (!column.isGenerated || column.type === "uuid")` and will work same and be easier to read